accounts/admin.py

```diff
@@ -4,7 +4,7 @@
 
 
 class StudentAdmin(admin.ModelAdmin):
-    readonly_fields = ("user",)
+    autocomplete_fields = ["user"]
```

Comment:
Why is this being changed?

Reply:
because otherwise you can't create student objects through admin, which is something that's useful to do

Reply:
needs to happen in platform-dev and in development, for example

Reply:
not directly a part of this PR but it was useful for testing

Reply:
Maybe it helps in development, but this very likely shouldn't be set in production since every student user who logs in will have a student model generated for them. Maybe we could help mitigate that with a postsave.

Reply:
yeah, should probably be a postsave then